content/millennium/r4/financial/account.md

```diff
@@ -86,7 +86,7 @@ Notes:
 
 * You may search via:
   * `_id` alone
-  * `patient`, `identifier`, and `type` set to 'statement'.
+  * `patient`, `identifier` system set to codeSet 28200, and `type` set to 'statement'.
```

Comment:
let's do it this way

```
* You may search via:
  * `_id` alone
  * `patient`, `identifier`, and `type` set to 'statement'
  * `-guarantor` and `type` set to 'financial-account'
* When searching via `identifier`, the system must be `https://fhir.cerner.com/<EHR source id>/codeSet/28200`. You may not search via `identifier` with a system of `https://fhir.cerner.com/accountnumber`
* The `-guarantor` search parameter should contain a reference to a RelatedPerson when set.
```

Reply:
Updated:
<img width="639" alt="Screen Shot 2020-06-29 at 3 05 50 PM" src="https://user-images.githubusercontent.com/27974425/86050963-3fc4ce80-ba1a-11ea-83fe-e96456dbc69d.png">